cpp/server/ct-server.cc

```diff
@@ -244,13 +246,20 @@ int main(int argc, char* argv[]) {
           ? new FakeEtcdClient(event_base)
           : new EtcdClient(event_base, FLAGS_etcd_host, FLAGS_etcd_port));
 
+  // No real reason to let this be configurable per node; you can really
+  // shoot yourself in the foot that way by effectively running multiple
+  // distinct elections.
+  const string kLockDir("/election");
```

Comment:
It's more about providing documentation - the variable name gives a local clue as to what that value is used for, and provides somewhere to anchor the comment which is intended to stop someone from helpfully changing a 'bad hardcoded constant' into a 'good FLAG' at some point in the future.